From 8a02fcf497d56dd9547d07590f6604e2a5b4dcda Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:53:08 -0400
Subject: [fs] jbd2: remove debug dependency on debug_fs and update Kconfig help text

Message-id: <1414058125-4183-35-git-send-email-lczerner@redhat.com>
Patchwork-id: 98126
O-Subject: [RHEL 7.1 PATCH v2 034/171] jbd2: remove debug dependency on debug_fs and update Kconfig help text
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Paul Gortmaker <paul.gortmaker@windriver.com>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit 75497d0607b56e16e4fec3a62ef447c752428723

 Commit b6e96d0067d8 ("jbd2: use module parameters instead of debugfs
 for jbd_debug") removed any need for a dependency on DEBUG_FS.  It
 also moved the /sys variables out from underneath the typical debugfs
 mount point.  Delete the dependency and update the /sys path to where
 the debug settings are currently.

 Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/jbd2/Kconfig b/fs/jbd2/Kconfig
index 69a48c2..5a9f553 100644
--- a/fs/jbd2/Kconfig
+++ b/fs/jbd2/Kconfig
@@ -20,7 +20,7 @@ config JBD2
 
 config JBD2_DEBUG
  bool "JBD2 (ext4) debugging support"
- depends on JBD2 && DEBUG_FS
+ depends on JBD2
  help
    If you are using the ext4 journaled file system (or
    potentially any other filesystem/device using JBD2), this option
@@ -29,7 +29,7 @@ config JBD2_DEBUG
    By default, the debugging output will be turned off.
 
    If you select Y here, then you will be able to turn on debugging
-   with "echo N > /sys/kernel/debug/jbd2/jbd2-debug", where N is a
+   with "echo N > /sys/module/jbd2/parameters/jbd2_debug", where N is a
    number between 1 and 5. The higher the number, the more debugging
    output is generated.  To turn debugging off again, do
-   "echo 0 > /sys/kernel/debug/jbd2/jbd2-debug".
+   "echo 0 > /sys/module/jbd2/parameters/jbd2_debug".
-- 
1.7.1